src/fragments/warehouses.ts

```diff
@@ -12,6 +12,8 @@ export const warehouseWithShippingFragment = gql`
   ${warehouseFragment}
   fragment WarehouseWithShippingFragment on Warehouse {
     ...WarehouseFragment
+    isPrivate
+    clickAndCollectOption
```

Comment:
I'm not sure if we need it here, especially having these fields defined in `WarehouseDetailsFragment`

Reply:
Done.